dask/array/tests/test_fft.py

```diff
@@ -197,14 +197,18 @@ def test_wrap_fftns(modname, funcname, dtype):
     )
 
 
-@pytest.mark.parametrize("funcname", ["fftfreq", "rfftfreq"])
 @pytest.mark.parametrize("n", [1, 2, 3, 6, 7])
 @pytest.mark.parametrize("d", [1.0, 0.5, 2 * np.pi])
-def test_fftfreq(funcname, n, d):
-    np_func = getattr(np.fft, funcname)
-    da_func = getattr(da.fft, funcname)
+def test_fftfreq(n, d):
+    assert_eq(da.fft.fftfreq(n, d, chunks=((n,),)), np.fft.fftfreq(n, d))
 
-    assert_eq(da_func(n, d, chunks=n), np_func(n, d))
+
+@pytest.mark.parametrize("n", [1, 2, 3, 6, 7])
+@pytest.mark.parametrize("d", [1.0, 0.5, 2 * np.pi])
+def test_rfftfreq(n, d):
+    assert_eq(
+        da.fft.rfftfreq(n, d, chunks=((n // 2 + 1,),)), np.fft.rfftfreq(n, d)
```

Comment:
A few things:

- There should be no reason to specify chunks as a tuple of tuples. Same goes for the `fftfreq` test above. Would be good to test the following:
  - `chunks=n//2 + 1`
  - `chunks=(1, 3)  # some tuple of ints that sum to proper length (in this case for n=7)`
- Formatting nit, please write as:
 
```
assert_eq(da.fft.rfftfreq(n, d, chunks=...),
          np.fft.rfftfreq(n, d))
```